packages/contracts-ts/wagmi.config.ts

```diff
@@ -0,0 +1,365 @@
+import { ContractConfig, defineConfig, Plugin } from '@wagmi/cli'
+import { actions, react } from '@wagmi/cli/plugins'
+import glob from 'glob'
+import { readFileSync, writeFileSync } from 'fs'
+import type { Abi, AbiFunction, Address } from 'abitype'
+import { isDeepStrictEqual } from 'util'
+
+/**
+ * Predeployed contract addresses
+ * In future it would be nice to have a json file in contracts bedrock be generated as source of truth
```

Comment:
+1 on this - splitting this metadata out from the contracts-bedrock source adds more work on contracts-bedrock maintainers

Reply:
Yea I'll figure this out in a follow up pr today that will also try to add the missing contract addresses